contrib/minizip/unzip.c

```diff
@@ -112,7 +110,7 @@
 # define ALLOC(size) (malloc(size))
 #endif
 #ifndef TRYFREE
-# define TRYFREE(p) { free(p);}
+# define TRYFREE(p) { free(p); }
```

Comment:
You could get rid of `TRYFREE` altogether. This is shorter and easier to read and understand:
```c
    free(p);
```
than:
```c
    TRYFREE(p);
```